control/control_loop_test.go

```diff
@@ -246,6 +246,7 @@ func BenchmarkLoop100(b *testing.B) {
 }
 
 func TestControlLoop(t *testing.T) {
+	t.Skip()
```

Comment:
Maybe put in a comment explaining why we're skipping this test and what needs to change before we stop skipping it.